src/Catel.Tests/Configuration/ConfigurationServiceFacts.serialization.cs

```diff
@@ -32,7 +32,7 @@ public SerializationConfigurationService()
                 {
                 }
 
-                protected override void SetValueToStore(ConfigurationContainer container, string key, string value)
+                protected override void SetValueToStore(ConfigurationContainer container, string key, object value)
```

Comment:
If the object can return null, I think this should be nullable as well.

Reply:
See [eff46b3](https://github.com/Catel/Catel/pull/2214/commits/eff46b3ca62d3eb927c945a6f30132d3fa124b77)